ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ScanConfigWizardForm.tsx

```diff
@@ -38,11 +39,17 @@ function ScanConfigWizardForm({ initialFormValues }: ScanConfigWizardFormProps):
     const formik = useFormikScanConfig(initialFormValues);
     const [isCreating, setIsCreating] = useState(false);
     const [createScanConfigError, setCreateScanConfigError] = useState('');
+    const [clustersUsedForProfileData, setClustersUsedForProfileData] = useState<string[]>([]);
 
     const listClustersQuery = useCallback(() => listComplianceIntegrations(), []);
     const { data: clusters, loading: isFetchingClusters } = useRestQuery(listClustersQuery);
 
-    const listProfilesQuery = useCallback(() => listComplianceProfiles(), []);
+    const listProfilesQuery = useCallback(() => {
+        if (clustersUsedForProfileData.length > 0) {
```

Comment:
Stylistic question for my understanding what are pro and con of `>` versus `!== 0` for `length` which cannot have a negative value because controlled by JavaScript.